modules/cells/src/main/java/dmg/util/logback/FilterThresholdSet.java

```diff
@@ -93,7 +93,7 @@ public synchronized void addAppender(String name) {
      */
     public synchronized Collection<String> getAppenders() {
         if (_parent == null) {
-            return Lists.newArrayList(_appenders);
+            return new ArrayList<>(_appenders);
```

Comment:
and so on :)

Reply:
Not doable in this case. As they cascade in the children and add elements as they do so.